enterprise/cluster/src/main/java/org/neo4j/cluster/statemachine/StateTransitionLogger.java

```diff
@@ -53,7 +53,7 @@ public void stateTransition( StateTransition transition )
     {
         Log log = logProvider.getLog( transition.getOldState().getClass() );
 
-        if ( log.isDebugEnabled() )
+//        if ( log.isDebugEnabled() )
```

Comment:
Why is this commented out? The rational was, as far as i understand it, to not create that big string message to log if it was going to be discarded anyway. Is the runtime cost assumption invalidated somehow? If that's the case, then we should remove the block altogether.